ampligraph/latent_features/models/EmbeddingModel.py

```diff
@@ -1169,13 +1177,19 @@ def fit(self, X, early_stopping=False, early_stopping_params={}, focusE_numeric_
                         pass
 
                     if self._perform_early_stopping_test(epoch):
+                        if tensorboard:
+                            self.writer.flush()
```

Comment:
I would suggest plotting validation loss, and validation mrr as well (in perform_early_stopping_test)

Reply:
Validation loss can't be added in this release as it is batch number dependent, will be added in AmpliGraph2. Criteria for early stopping added to the tracking.